go/vt/sqlparser/sql.y

```diff
@@ -3609,6 +3609,8 @@ value_expression:
   {
     if num, ok := $2.(*Literal); ok && num.Type == IntVal {
```

Comment:
Could we move these methods out from `sql.y`? I think it's nice if the code in there is as simple as possible.

Reply:
done